features/bootstrap/GraphqlContext.php

```diff
@@ -97,6 +98,48 @@ public function ISendTheGraphqlRequestWithOperation(string $operation)
         $this->sendGraphqlRequest();
     }
 
+    /**
+     * @Given I have the following files for a GraphQL request:
+     */
+    public function iHaveTheFollowingFilesForAGraphqlRequest(TableNode $table)
+    {
+        $files = [];
+
+        foreach ($table->getHash() as $row) {
+            if (!isset($row['name']) || !isset($row['file'])) {
+                throw new \Exception("You must provide a 'name' and 'file' column in your table node.");
+            }
+
+            if (is_string($row['file'])) {
+                $files[$row['name']] = $this->restContext->getMinkParameter('files_path').DIRECTORY_SEPARATOR.$row['file'];
+            }
+        }
+
+        $this->graphqlRequest['files'] = $files;
+    }
+
+    /**
+     * @Given I have the following GraphQL form multipart map:
+     */
+    public function iHaveTheFollowingGraphqlFormMultipartMap(PyStringNode $string)
+    {
+        $this->graphqlRequest['map'] = $string->getRaw();
+    }
+
+    /**
+     * @When I send the following GraphQL form multipart operations:
+     */
+    public function iSendTheFollowingGraphqlFormMultipartOperations(PyStringNode $string)
+    {
+        $params = [];
+        $params['operations'] = $string->getRaw();
+        $params['map'] = $this->graphqlRequest['map'];
+
+        $this->request->setHttpHeader('Content-type', 'multipart/form-data');
+        $this->request->setHttpHeader('Accept', null);
```

Comment:
Is this necessary?

Reply:
No, Because we already had it here:
https://github.com/api-platform/core/blob/master/features/bootstrap/GraphqlContext.php#L125
I added this `Accept` header.